Update s3_to_exasol to python3

mjohannfunke opened the pull request:
I noticed that the s3_to_exasol script still uses Python 2, which is generally discontinued in 2020 and will be discontinued by exasol in the next major release. I updated the code to Python 3 and their respective packages.
As far as I can tell this now provides the same functionality, it lists all keys in a given bucket (using the ContinuationToken logic of boto3).

Since I am handling json files right now, I cannot test the remaining parts of the script (importing csvs).

Commit:
Update s3_to_exasol to python3

allipatev commented:
Hi @mjohannfunke, thanks for your contribution!
Let us take a look.
I was also thinking of leaving out the "force_http" input parameter as it doesn't seem applicable to boto3.

Commit:
Remove force_http parameter

mjohannfunke commented:
Hey @allipatev ,
you are right with the force_http parameter, I removed it. I also looked through the whole script to see where it was used and removed those spots as far as I can tell.
